cmd/query/app/builder/storage.go

```diff
@@ -33,7 +33,7 @@ import (
 // StorageBuilder is the interface that provides the necessary store readers
 type StorageBuilder interface {
 	NewSpanReader() (spanstore.Reader, error)
-	NewDependencyReader() (dependencystore.Reader, error)
+	NewDependencyReader(tableName string) (dependencystore.Reader, error)
```

Comment:
I don't believe the Builder needs this argument. It can get it and store internally from the config.